inst/app/ui.R

```diff
@@ -170,6 +173,59 @@ ui <- function(request) { page_navbar(
     )
   ),
 
+  tabPanel(
+    title = 'Bootstrap',
+    value = 'bootstrap',
+    layout_sidebar(
+      sidebar = sidebar(
+        width = sidebar_width,
+        accordion(
+          open = NULL, multiple = TRUE, class = 'mb-2',
+          accordion_panel(
+            'Bootstrap options',
+            selectizeInput(
+              'boot_rts', 'Returns to scale', choices = vals$rts, selected = 'crs'
```

Comment:
DRS/IRS is not available for Bootstrap, but they still appear in the dropdown-menu. Maybe create a separate vals-list with just CRS/VRS?

Reply:
They are included, as they mirror the RTS selected for DEA. Removing them will lead to an error, or even worse, an endless loop, as the inputs update each other (yes, I tried).

Reply:
Got it. Let's leave them then. Not sure about this integration though, see comment below.